spec/declaration.dd

```diff
@@ -90,6 +90,7 @@ $(GNAME StorageClasses):
 $(GNAME StorageClass):
 $(MULTICOLS 5,     $(GLINK2 attribute, LinkageAttribute)
     $(GLINK2 attribute, AlignAttribute)
+    $(GLINK2 attribute, AtAttribute)
```

Comment:
Fine for now, but I don't think that this (and `align`) belongs into `StorageClass`.
They should probably be extracted into a different rule and combined a few layers above.

Reply:
Yes, attributes and storage classes are somewhat of a mess that will probably not be easy to untangle.